packages/react/src/useSignAndSendTransaction.ts

```diff
@@ -0,0 +1,90 @@
+import {
+    SolanaSignAndSendTransaction,
+    SolanaSignAndSendTransactionFeature,
+    SolanaSignAndSendTransactionInput,
+    SolanaSignAndSendTransactionOutput,
+} from '@solana/wallet-standard-features';
+import {
+    WALLET_STANDARD_ERROR__FEATURES__WALLET_ACCOUNT_CHAIN_UNSUPPORTED,
+    WalletStandardError,
+} from '@wallet-standard/errors';
+import { getWalletAccountFeature, type UiWalletAccount } from '@wallet-standard/ui';
+import { getWalletAccountForUiWalletAccount_DO_NOT_USE_OR_YOU_WILL_BE_FIRED } from '@wallet-standard/ui-registry';
+import { useCallback } from 'react';
+
+import { OnlySolanaChains } from './chain';
+
+type Input = Readonly<
+    Omit<SolanaSignAndSendTransactionInput, 'account' | 'chain' | 'options'> & {
+        options?: Readonly<{
+            minContextSlot?: bigint;
+        }>;
```

Comment:
Wait, this makes no sense. These are per-transaction-send level options, not wallet-account-feature level options.